rental.rb

```diff
@@ -0,0 +1,15 @@
+class Rental
+  attr_accessor :date
+  attr_reader :person, :book
+
+  def initialize(book, person, date)
+    @person = person
+    @date = date
```

Comment:
[OPTIONAL] The `@date` instance variable should be initialized in the constructor.